snippets/get_transaction.py

```diff
@@ -0,0 +1,31 @@
+"""A snippet that walks us through getting a transaction."""
+from xrpl.clients import JsonRpcClient
+from xrpl.models.requests import Ledger, Tx
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234/")
+
+ledger_request = Ledger(transactions=True, ledger_index="validated")
+ledger_response = client.request(ledger_request)
+print(ledger_response)
+
+transactions = ledger_response.result["ledger"]["transactions"]
+
+if transactions:
```

Comment:
What happens if there are no transactions? (Ex. Will often close a ledger with no transactions)
https://devnet.xrpl.org/

Reply:
This specific snippet should probably be on mainnet, then - otherwise you'll mostly get nothing